python/ray/serve/handle.py

```diff
@@ -104,13 +105,27 @@ def __init__(
 
         self.router: Router = _router or self._make_router()
 
+        self._stop_event = threading.Event()
+        self._pusher = start_metrics_pusher(
+            interval_s=10,  # Queue handle metrcis
```

Comment:
Is this interval too large? On average, does this means the cold start delay for autoscaling increases by 5 seconds?

Reply:
that is correct,  change it to use config number.